aidial_adapter_bedrock/server/exceptions.py

```diff
@@ -34,6 +34,20 @@ def get_exception_type(status_code: int) -> str:
     return "internal_server_error"
 
 
+def _get_meta_status_code(response: dict) -> int | None:
+    code = response.get("ResponseMetadata", {}).get("HTTPStatusCode")
+    if isinstance(code, int):
+        return code
+    return None
+
+
+def _get_response_error_code(response: dict) -> int | None:
+    code = response.get("Error", {}).get("Code")
+    if isinstance(code, str) and code.lower() == "throttlingException".lower():
```

Comment:
We could also add 
[modelTimeoutException](https://docs.aws.amazon.com/bedrock/latest/APIReference/API_runtime_InvokeModelWithResponseStream.html#API_runtime_InvokeModelWithResponseStream_ResponseSyntax) error code handling

Also, maybe let's wrap it into some enum like this
```python
class BedrockExceptionCode(Enum):
    THROTTLING = "throttlingException"
    MODEL_TIMEOUT = "modelTimeoutException"

    def __eq__(self, other):
        if isinstance(other, str):
            return self.value.lower() == other.lower()
        return super().__eq__(other)
```

This would allow us to use match/case here
```python

match code:
    case BedrockExceptionCode.THROTTLING:
        return 429
    case BedrockExceptionCode.MODEL_TIMEOUT:
        return 408
    case _:
        return None
```
What do you think?

Reply:
Indeed, fixed